src/library_idbfs.js

```diff
@@ -15,16 +15,56 @@ mergeInto(LibraryManager.library, {
       // reuse all of the core MEMFS functionality
       return MEMFS.mount.apply(null, arguments);
     },
+#if IDBFS_DEFER_WRITE
+    DEFER_WRITE_TIMEOUT: 10000,
+    deferredSyncFs: function(mount) {
+      // If there is already a timer, don't set another one.
+      if (mount.mountSyncfsTimer) {
+        return;
+      }
+
+      // Set a timer what would trigger syncfs after 10 sec.
+      mount.mountSyncfsTimer = setTimeout(function() {
```

Comment:
Best to use `Browser.safeSetTimeout`, which will avoid running the timeout if the application exited already.